### PR TITLE
(fix) Force CSV data to be UTF-8 encoded

### PR DESCRIPTION
**Pull request by davidwinter**
## Trello card URL:

https://trello.com/c/NNPfKgvv/872-handle-invalid-characters-when-uploading-bulk-invites

## Changes in this PR:

* Forces encoding of CSV data to UTF-8

A combination of things going on with this. It seems like the line encodings in the file are misleading the CSV parser into the incorrect encoding type. That means that some whitespace after the first email address in the fixture file is not detected correctly as whitespace, and therefore not being stripped. It then fails the email validation. Then, when the email is being output in the error messages, the string is still appearing to be in an incorrect encoding.

Forcing the raw CSV data to UTF-8 resolves all of these issues.

### CODE REVIEW

**Comment by davidwinter** [comment]
Sorting a test for this currently, just took quite a bit of playing around to get something that worked with the troublesome file.

File: spec/fixtures/school_data_invalid_encoding.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+school_urn,given_name,family_name,email,school_name,trust_name,Notes for developers,do not use,do not use 2
+102893,Test,User,david@madetech.com​,​Test School,Trust Name,,,
+146850,Test,User,test@email.com,Another School,Trust Name,,,
```
Comment (erbridge) [inline spec/fixtures/school_data_invalid_encoding.csv]: Do we have any characters in here that don't translate to `utf-8`?

Comment (davidwinter) [reply spec/fixtures/school_data_invalid_encoding.csv]: There are some invisible characters after the email addresses that are stripped out once we have the force encoding of UTF-8 in place.

Comment (erbridge) [reply spec/fixtures/school_data_invalid_encoding.csv]: 👍

**Pull request merged**